R/CensoredDataSuite.R

```diff
@@ -215,6 +215,15 @@ simpleCensoredMethods <- function(.data, nd_method = "multiplier", nd_multiplier
 #' 
 #' @export
 #' 
+#' @examples
+#' # Load example dataset:
+#' data("TADAProfileClean18_TNonly")
+#' # TADAProfileClean18_TNonly dataframe is clean, harmonized, and filtered
+#' # down to one Comparable Data Identifier
+#' 
+#' # Create summarizeCensoredData table:
+#' TADAProfileClean18_TNonly_summarizeCensoredData <- summarizeCensoredData(TADAProfileClean18_TNonly)
```

Comment:
Consider including counts for NA, NA or ND, and text or other data types that data users should be aware of in the table. These should all be 0 before moving on in the assessment process... consider including option to autoclean as part of this function (if any of the above are not 0, then automatically deal with them, i.e. remove/flag for removal)